test/shop.spec.js

```diff
@@ -989,6 +989,30 @@
                             expect(eventListener.setCouponCode).toHaveBeenCalledWith('coupon-code');
                         });
                     });
+
+                    describe('and there are two payment methods - "wire-transfer" and "paypal-classic"', function() {
+                      beforeEach(function () {
+                          function GatewaySpy() {
+                              this.fetchApplicationProfile = function (request, response) {
+                                  response.success({
+                                      availablePaymentMethods: ['wire-transfer', 'paypal-classic']
+                                  });
+                              };
+                          }
+
+                          binarta.application.gateway = new GatewaySpy();
+                          binarta.application.refresh();
+                          binarta.shop.checkout.start(order, [
+                              'summary',
+                              'completed'
+                          ]);
+                      });
+
+                      it('paypal is selected by default', function () {
+                        expect(binarta.shop.checkout.getPaymentProvider()).toBe('paypal-classic');
+                      });
+
```

Comment:
Please clear out this blank line. I know it's not a big thing but I know I will delete it when I see it and that will cause a diff to appear on other commits distracting from the actual changes intended for that commit.